### ISSUE TITLE
Question, any plans for...

### ISSUE THREAD

**Issue by YukiSakuma** [ai-forever/ru-dalle#121]
A ru-dalle2? If yes any timelines? Thanks for answering

**Comment by XieBaoshi** [comment]
You might want to take a look at the Github project "DALL-E 2 Pytorch", which is exactly an open source DALL-E 2 (WIP).

**Comment by YukiSakuma** [comment]
> You might want to take a look at the Github project "DALL-E 2 Pytorch", which is exactly an open source DALL-E 2 (WIP).

Yes, I'm aware of it, I check it almost everyday for updates, so I'm curious if a ru-dalle2 is being worked on.

**Comment by NastyaMittseva** [comment]
We have realized the multilingual latent diffusion model Kandinsky-2.0,  the code and a detailed description can be found at https://github.com/ai-forever/Kandinsky-2.0.
